ground/src/test/java/com/google/android/ground/persistence/remote/firebase/schema/SubmissionLocalDataStoreConverterTest.kt

```diff
@@ -89,22 +89,18 @@ class SubmissionLocalDataStoreConverterTest {
           AUDIT_INFO_2,
           TaskDataMap(
             mapOf(
-              Pair("task1", TextTaskData("Text taskData")),
-              Pair(
-                "task2",
+              "task1" to TextTaskData("Text taskData"),
+              "task2" to
```

Comment:
I'm not a huge fan of pairing the multiline calls using `to` syntax -- I think it's fine for the simple values, though -- not something we should do anything about right now, just a note for later.

Reply:
This was a manual change and isn't suggested by the plugin. Reverted it for now to avoid the confusion. But thanks for pointing it out.